components/automate-deployment/pkg/converge/compiler_test.go

```diff
@@ -75,7 +75,7 @@ func NoSupervisorUpgradeRequired(m *target.MockTarget) *target.MockTarget {
 	m.On("RenderAutomateUnitFile", "", testHabBinPkg, testHabLauncherPkg).Return(string(testAutomateUnitFileContent), nil)
```

Comment:
this would be a good time to let this file die

Reply:
I require no convincing